ccat.py

```diff
@@ -105,15 +126,42 @@ def run_module(self, answers):
             data = ecr__push_repos.main(cli_answers)
             self.print_module_summary(data, ecr__push_repos)
 
+        elif SWAP_AWS_PROFILE in answers['main_menu']:
+            self.extentions['aws'].swap_profile()
+        
+        # GCP
+        elif ENUMERATE_GCR in answers['main_menu']:
+            cli_answers = self.extentions['gcp'].ask_gcr_enum_repos()
+            self.print_module_running(gcr__enum_repos.module_info['name'])
+            data = gcr__enum_repos.main(cli_answers)
+            self.extentions['gcp'].data.update({'gcr_repos': data})
+            self.print_module_summary(data, gcr__enum_repos)
+
+        elif LIST_GCR_REPOS in answers['main_menu']:
+            self.extentions['gcp'].print_gcr_repos()
+
+        elif PULL_GCR_REPOS in answers['main_menu']:
+            cli_answers = self.extentions['gcp'].ask_gcr_pull_repos()
+            self.print_module_running(gcr__pull_repos.module_info['name'])
+            data = gcr__pull_repos.main(cli_answers)
+            self.print_module_summary(data, gcr__pull_repos)
+
+        elif PUSH_GCR_REPOS in answers['main_menu']:
+            cli_answers = self.extentions['gcp'].ask_gcr_push_repos()
+            self.print_module_running(gcr__push_repos.module_info['name'])
+            data = gcr__push_repos.main(cli_answers)
+            self.print_module_summary(data, gcr__push_repos)
+
+        elif SWAP_SERVICE_ACCOUNT in answers['main_menu']:
+            self.extentions['gcp'].swap_service_account()
```

Comment:
Or  access token right?

Reply:
Mm, I need to change the name of that variable to SWAP_GCP_CREDS b/c we swap both service account and access token.